docs/openhabian-DEBUG.md

```diff
@@ -6,88 +6,204 @@ source: https://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.
-
-Next, you need your router (or a different device) to provide properly configured DHCP services so your openHABian box gets an IP address assigned when you boot it for the first time.
-The DHCP server also has to announce which DNS resolver to use so your box knows how to translate DNS names into IP addresses.
-It also needs to announce which IP address to use as the default gateway to the internet - a typical access router is also the DHCP server will announce it's own address here.
-Finally, the DHCP server should also announce the NTP server(s) to use for proper time services. Lack thereof will not break the installation procedure but can lead to all sorts of long term issues so we recommend to setup DHCP to announce a reachable and working NTP server.
-
-A note on IPv6: openHABian was reported failing to boot in some environments that make use of IPv6. If basic IP initialization fails (you cannot `ping` your box) or installation gets stuck trying to download software packages, you might want to try disabling IPv6. You can also do that before the very first install attempt if you're sure you don't need any IPv6 connectivity on your openHABian box. See [this section of openhabian.md](openhabian.md#ipv6-notes) how to disable IPv6 on your system.
-Note that this is just a summary to cover the most commonly encountered cases. The full boot procedure and how to obtain IP addresses, DNS resolver, default route and NTP server addresses are highly complex and widely customizable and a comprehensive description on how to properly configure your Internet access and router are out of scope of openHABian. Please Google for how to accomplish that.
+It may work to install and run openHABian on unsupported hardware. If it does
+not work, you are welcome to find out what's missing and contribute it back to
+the community with a Pull Request. It is sometimes simple things like a naming
+string. We'll be happy to include that in openHABian so you can use your box
+with openHABian.
+We'll keep that code in unless there's a valid reason to change or remove it.
+However, that does not make your box a "supported" one as we don't have it
+available for our further development and testing. So there remains a risk that
+future openHABian releases will fail to work on your SBC because we changed a
+thing that broke support for your HW - unintentionally however inevitably.
+
+openHABian requires you to provide direct Internet access. Using private IP
+addresses is fine as long as your router properly provides NAT
+(Network Address Translation) services.
+Note: we assume an Ethernet connection to be present at time of installation.
+We do not support installing via WiFi.
```

Comment:
I think that's wrong, we *do* support wifi-only. Didn't know by the time I wrote it.
Not sure how to consistently write that down.
Maybe add a sort of flowchart with a section "generic IP connectivity" at the start, then inside that section tell users to *try* with Ethernet when they only tried Wifi ?

Reply:
Dunno how to make a flow chart in MarkDown but is that better?

Reply:
me neither. Yes that's ok.